tests/pools/test_pool_rpc.py

```diff
@@ -1066,125 +1066,3 @@ async def status_is_farming_to_pool_no_blocks():
             client.close()
             await client.await_closed()
             await rpc_cleanup()
-
```

Comment:
Why are we removing this test? It would be nice to have this passing.

Reply:
it was very slow on CI (1000 blocks). This test was added here.